README.md

```diff
@@ -1,34 +1,34 @@
 # Multi-Cloud Platform Foundations
 
-* [Mesoform Multi-Cloud Platform Foundatuins](#mesoform-multi-cloud-platform-foundations)
-  * [Information](#Information)
+* [Mesoform Multi-Cloud Platform Foundatuins](#multi-cloud-platform-foundations)
+  * [Background](#Background)
 * [This Repository](#this-repository)
-  * [Quick start guide](#quick-start-guide)
-* [MMCF](#MMCF)
+  * [MCCF](#MCCF)
 * [Contributing](#Contributing)
 * [License](#License)  
 
-## Information
-Mesoform Multi-Cloud Platform (MCP) is a concept, and supporting infrastructure code which simplifies the deployment of
-applications across multiple Cloud providers. The basis behind MCP is for platform engineers and application
+## Background
+Mesoform Multi-Cloud Platform (MCP) is a set of tools and supporting infrastructure code which simplifies the deployment
+of applications across multiple Cloud providers. The basis behind MCP is for platform engineers and application
 engineers to be working with a single structure and configuration for deploying foundational infrastructure (like IAM
 policies, Google App Engine or Kubernetes clusters) as would be used for deploying workloads to that infrastructure
 (e.g. Containers/Pods).
 
-Within this framework is a unified configuration language called Mesoform Multi-Cloud Configuration Format (or MMCF),
+Within this framework is a unified configuration language called Mesoform Multi-Cloud Configuration Format (or MCCF),
 which is detailed below and provides a familiar YAML structure to what many of the native original services offer and
 adapts it into HCL, the language of Hashicorp Terraform, and deploys it using Terraform, to gain the benefits (like
 state management) which Terraform offers.
```

Comment:
This section is identical to the [one in the other repository](https://github.com/mesoform/Multi-Cloud-Platform/blob/master/README.md?plain=1#L18-L28). Would it be appropriate to replace the duplication with an appropriate link?